app/repos/notification-log.js

```diff
@@ -20,4 +20,39 @@ const logRejectedNotification = async (message, notifyError) => {
   })
 }
 
-export { logCreatedNotification, logRejectedNotification }
+const getPendingNotifications = async () => {
+  const pending = await db.notifyApiRequestSuccess.findAll({
```

Comment:
I don't see findAll causing any issue now, but may be by the time we have many subscribers with thousands of request, so, we may need to fetch records in batches and not just one go, when fetching all at one go becomes a pain.

Reply:
Good point. I suppose it depends on how many requests we're expecting per minute. Might be something for us to find out and define.